src/style.css

```
@@ -0,0 +1,118 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+}

```

```suggestion
* {
  box-sizing: border-box;
}
```
Don't reset paddings using * selector. It applies to all the elements, which is not good for performance